### PR TITLE
fix(rds): restrict RDS inbound access by removing open IP rule

### PR DESCRIPTION
**Pull request by nitesh-balla**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -295,6 +295,8 @@ while true; do
 
 done
 
+export AWS_DEFAULT_REGION;
```
Comment (jagan-jaya) [inline install.sh:298]: why is this required?

File: lib/aws/stack.ts [hunk 2]
```diff
@@ -25,9 +25,10 @@ export class AWSStack extends cdk.Stack {
     let vpc = new Vpc(this, config.vpc);
     let subnets = new SubnetStack(this, vpc.vpc, config);
     let elasticache = new ElasticacheStack(this, config, vpc.vpc);
-    let rds = new DataBaseConstruct(this, config.rds, vpc.vpc, isStandalone );
-    rds.sg.addIngressRule(ec2.Peer.ipv4("0.0.0.0/0"), ec2.Port.tcp(5432)); // this has to be moved to standalone and for production it should be internal jump
-
+    let rds = new DataBaseConstruct(this, config.rds, vpc.vpc, isStandalone);
+    if (isStandalone) {
+      rds.sg.addIngressRule(ec2.Peer.ipv4("0.0.0.0/0"), ec2.Port.tcp(5432)); // this has to be moved to standalone and for production it should be internal jump
```
Comment (jagan-jaya) [inline lib/aws/stack.ts:30]: we need to remove in other places also right?

File: install.sh (hunk 1 above)
Comment (nitesh-balla) [reply install.sh:298]: In few cases if `AWS_DEFAULT_REGION` is not exported by the user. CDK is not populating `CDK_DEFAULT_REGION`. So, just to ignore the case where the user didn't export `AWS_DEFAULT_REGION` I am doing it in the script.

File: lib/aws/stack.ts (hunk 2 above)
Comment (nitesh-balla) [reply lib/aws/stack.ts:30]: As this PR RDS Specific, Removing here would suffice. I have created a separate issue for restricting external jump and internal jump public access.

**Review by jagan-jaya — APPROVED** [review]
